src/components/common/Header/styles.module.css

```diff
@@ -28,7 +28,11 @@
 .logo svg {
   width: auto;
   display: block;
-  color: var(--color-logo-main);
+  color: '#000';
+}
+
+[data-theme='dark'] .logo svg {
+  color: '#fff';
```

Comment:
Why `#fff`?

Reply:
The text colours will change with the upcoming branding.